packages/lesswrong/server/repos/CommentsRepo.ts

```diff
@@ -9,6 +9,7 @@ import { EA_FORUM_COMMUNITY_TOPIC_ID } from "../../lib/collections/tags/collecti
 type ExtendedCommentWithReactions = DbComment & {
   yourVote?: string,
   theirVote?: string,
+  userVote?: string,
```

Comment:
I think this doesn't belong on this type - or rather this type shouldn't be used as the return value for `getPopularPollCommentsWithUserVotes`, since it will never return `yourVote` or `theirVote`.